docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -1,116 +1,252 @@
+
+      .. code-block:: cmd
+
+         # configures OneAPI environment variables
+         call "C:\Program Files (x86)\Intel\oneAPI\setvars.bat"
+
+         set SYCL_CACHE_PERSISTENT=1
+
+      Please also set the following environment variable for iGPU:
+
+      .. code-block:: cmd
+
+         set BIGDL_LLM_XMX_DISABLED=1
+
+      .. note::
+
+         For the first time that **each** model runs on **a new machine**, it may take around several minutes to compile.
 
-   Compatible versions of ``torch`` and ``torchvision`` are prerequisites for installing ``intel_extension_for_pytorch``.
 ```
 
-Then you may install directly from the wheel archives using following commands:
+### 3. How to install from wheel
 
-```bash
-# install the packages from the wheels
-python -m pip install torch-2.0.1a0+cxx11.abi-cp39-cp39-linux_x86_64.whl
-python -m pip install torchvision-0.15.2a0+cxx11.abi-cp39-cp39-linux_x86_64.whl
-python -m pip install intel_extension_for_pytorch-2.0.110+xpu-cp39-cp39-linux_x86_64.whl
+If you encounter network issues when installing IPEX, you can also install BigDL-LLM dependencies for Intel XPU from source achieves. First you need to install the target torch/torchvision/ipex versions from downloaded wheels [here](https://developer.intel.com/ipex-whl-stable-xpu) before installing `bigdl-llm`. 
+
+```eval_rst
+.. tabs::
+   .. tab:: PyTorch 2.1 Linux
+
+      .. code-block:: bash
+
+         # get the wheels on Linux system for IPEX 2.1.10+xpu
+         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torch-2.1.0a0%2Bcxx11.abi-cp39-cp39-linux_x86_64.whl
+         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torchvision-0.16.0a0%2Bcxx11.abi-cp39-cp39-linux_x86_64.whl
+         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/intel_extension_for_pytorch-2.1.10%2Bxpu-cp39-cp39-linux_x86_64.whl
+
+      Then you may install directly from the wheel archives using following commands:
+
+      .. code-block:: bash
+
+         # install the packages from the wheels
+         pip install torch-2.1.0a0+cxx11.abi-cp39-cp39-linux_x86_64.whl
+         pip install torchvision-0.16.0a0+cxx11.abi-cp39-cp39-linux_x86_64.whl
+         pip install intel_extension_for_pytorch-2.1.10+xpu-cp39-cp39-linux_x86_64.whl
+
+         # install bigdl-llm for Intel GPU
+         pip install --pre --upgrade bigdl-llm[xpu_2.1]
+
+   .. tab:: PyTorch 2.1 Windows
+
+      .. code-block:: bash
+
+         # get the wheels on Windows system
+         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torch-2.1.0a0%2Bcxx11.abi-cp39-cp39-win_amd64.whl
+         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torchvision-0.16.0a0%2Bcxx11.abi-cp39-cp39-win_amd64.whl
+         wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/intel_extension_for_pytorch-2.1.10%2Bxpu-cp39-cp39-win_amd64.whl
```

Comment:
All wheels are for python=3.9, if we update above python versions, maybe also add comment here. Just a note, no need to change in this PR.